cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/ACLineSegmentConversion.java

```diff
@@ -110,6 +111,7 @@ public void convertMergedLinesAtNode(PropertyBag other, String boundaryNode) {
         } else {
             mline = createQuadripole(boundaryLine1, boundaryLine2);
         }
+        addAliases(mline);
```

Comment:
Missing addAliases for merged line when line-switch configuration is seen at boundary ?